README.md

```diff
@@ -1,3 +1,8 @@
+# Deprecated
+
+This project has been [deprecated](https://github.com/external-secrets/kubernetes-external-secrets/issues/864). 
+Please take a look at ESO (External Secrets Operator) instead https://github.com/external-secrets/external-secrets 
```

Comment:
Could we link to the Migration guide/tooling as well to direct consumers to the vetted method to switch?